### PR TITLE
Feature/mac 3189 filter panel

### PR DESCRIPTION
**Pull request by lucasfiltered**
# <JIRA issue / Title>
mac 3189 filter panel 
## TL;DR

-   List
-   The
-   Core
- Changes

## Details
draft PR
Checklist / Reminders:

-   [ ] Documentation / Story is created / adapted
-   [ ] Unit tests
-   [ ] CSS changes are BEM-compliant

### CODE REVIEW

**Review by tricaalexandru — DISMISSED** [review]

File: src/components/Organisms/FilterPanel/FilterPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+export interface IFilter {
+  title?: string
+}
+export type FilterType = 'default' | 'selected' | 'active'
+
+export const FilterPanel: React.FC<IFilter> = (props): JSX.Element => {
+  return (
+    <Modal
+      // {...modalKnobsWithTitle()}
```
Comment (tricaalexandru) [inline src/components/Organisms/FilterPanel/FilterPanel.tsx:17]: @lucasfiltered do we need this commented out code?

Comment (matthill8286) [reply src/components/Organisms/FilterPanel/FilterPanel.tsx:17]: @tricaalexandru this is copied directly from the `.story` file as you only get  knobs in those files

**Review by tricaalexandru — APPROVED** [review]

**Comment by tricaalexandru** [comment]
@lucasfiltered can you please also add some screenshots

File: src/components/Organisms/FilterPanel/FilterPanel.tsx (hunk 1 above)
Comment (lucasfiltered) [reply src/components/Organisms/FilterPanel/FilterPanel.tsx]: is removed in my next commit

**Pull request merged**